ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -138,6 +138,12 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-perfscale-e2e-configmap-scale
+- as: e2e-gcp-five-cp-nodes
```

Comment:
I'm not in love with this short name - let's not spread it elsewhere without more discussion (but it's fine here)

Reply:
Done